src/scripts/actions/UploadActions.js

```diff
@@ -47,7 +53,9 @@ export const upload = (file: Object) => (dispatch: Dispatch<*>) => {
     dispatch(uploadProgress({ id: newVideoId, progress: progressPercent }))
   })
   uploader.on('fileReady', function (file) {
-    dispatch(uploadLocalSuccess({ id: newVideoId, hash: file.hash }))
```

Comment:
Might want to set a default value for file just in _case_ it comes back `null/undefined`. Or just check that it is truthy before grabbing properties off of it to avoid a runtime error